tests/siuparser.test.js

```diff
@@ -0,0 +1,63 @@
+import { siuAxel, siuFede, siuExactas } from "./siu-files";
+import { parseSIU } from "../src/siuparser";
+
+const sius = [ siuAxel, siuFede, siuExactas ];
+
+describe.each(sius)("essential tests", (siuName, siuRawData, siuJSON) => {
+  const parsedSIU = parseSIU(siuRawData);
```

Comment:
Tal vez esto sirve para hacer menos verboso el testeo.

```suggestion
  console.debug = jest.fn(); // Deshabilitar el debugging que se usa en el browser
  const parsedSIU = parseSIU(siuRawData);
```

 `jest` se importa con `import { jest } from "@jest/globals";`